modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc

```diff
@@ -79,9 +79,45 @@ $ curl -L $ISO_URL -o rhcos-live.iso
 
 . Use the {op-system} ISO and the hosted `worker.ign` file to install the worker node:
 
-.. Boot the target host using the {op-system} ISO and your preferred method of installation.
+.. Boot the target host with the {op-system} ISO and your preferred method of installation.
 
-.. When the target host has booted from the {op-system} ISO, open a console on the target host, and run the following `coreos-installer` command to install the system:
+.. When the target host has booted from the {op-system} ISO, open a console on the target host.
+
+.. Optional: If your local network does not have DHCP enabled, you need to specify the static IP of the worker node before running the {op-system} installation. For example, run the following commands on the target host console:
+
+... Configure the worker host network connection with a static IP. For example, run the following command:
++
+[source,terminal]
+----
+$ nmcli con mod <network_interface> ipv4.method manual /
+ipv4.addresses <static_ip>/<cidr> ipv4.gateway <network_gateway> ipv4.dns <dns_server>
+----
++
+where:
++
+--
+<static_ip>:: Is the host static IP address, for example, `10.1.101.50`
+<network_gateway>:: Is the network gateway and IP address range, for example, `10.1.101.62`
+<cidr>:: Is the network IP address range, for example, `28`
+--
++
+Activate the modified network interface:
++
+[source,terminal]
+----
+$ nmcli con up <network_interface>
+----
++
+Run the `coreos-installer` passing in the static IP and DNS details:
++
+[source,terminal]
+----
+$ sudo coreos-installer install --firstboot-args="console=tty0 rd.neednet=1 /
+ip=<static_ip>::<network_gateway>:<cidr>en:none nameserver=<dns_server>" --copy-network /
+--ignition-url http://<url/worker.ign> /dev/sda --insecure-ignition
```

Comment:
/dev/sda is a specific hard disk name. Should be <hard disk>  and then followed by explanation:

<hard_disk>:: Is hard disk where you install {op-system}, for example, `/dev/sda`